packages/atomic/cypress/e2e/common-assertions.ts

```diff
@@ -72,6 +72,16 @@ export function assertContainsComponentError(
   });
 }
 
+export function assertContainsComponentErrorWithoutIt(
```

Comment:
I'm not sure I see the value of having these checks without the `it`... Would you mind elaborating on why this is necessary?